src/payment/AlertCodeMessages.jsx

```diff
@@ -65,46 +69,50 @@ EnrollmentCodeQuantityUpdated.propTypes = {
   }).isRequired,
 };
 
-export const TransactionDeclined = () => (
-  <>
+export function TransactionDeclined() {
+  return (
     <FormattedMessage
       id="payment.messages.transaction.declined.body"
       defaultMessage="Your payment could not be processed. Please check your payment information or reach out to your bank or financial institution for further assistance."
       description="Asks the user to check their information or contact their payment provider for help."
     />
-  </>
-);
+  );
+}
 
-export const CaptureKeyTimeoutExplanation = () => (
-  <>
+export function CaptureKeyTimeoutExplanation() {
+  return (
     <FormattedMessage
       id="payment.messages.key.timeout.body"
       defaultMessage="For security, your credit card information will then need to be re-entered to complete your purchase."
       description="Briefly explains the credit card field timeout."
     />
-  </>
-);
+  );
```

Comment:
Good catch on the unnecessary fragment - I'm surprised the linter didn't catch that before!

Reply:
Me too!